src/AssetsPage/AssetsFilters/index.jsx

```diff
@@ -21,7 +29,7 @@ const ASSET_TYPES = [
   },
 ];
 
-const AssetsFilters = ({ assetsParameters, updateFilter }) => (
+export const AssetsFilters = ({ assetsParameters, updateFilter }) => (
```

Comment:
This might be premature, but it'll be necessary when testing the AssetsFilter, since we want to test the unwrapped component. It actually was meant for a separate testing PR but ended up here. I'm happy to remove it if it doesn't make sense here.